downstream/modules/eda/proc-eda-set-up-new-decision-environment.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc-eda-set-up-new-decision-environment"]
+
+= Setting up a new decision environment
+
+Decision Environments are a container image to run Ansible rulebooks.
+They create a common language for communicating automation dependencies, and provide a standard way to build and distribute the automation environment.
+The default decision environment is found at link:https://quay.io/repository/ansible/ansible-rulebook[Ansible-Rulebook]. 
+
+To create your own decision environment refer to [PLACEHOLDER FOR LINK TO KCS].
+
+The following steps guide you on how to import a decision environment into your {EDAcontroller} Dashboard.
+
+== Prerequisites
+
+* You are logged in as a Content Consumer.
+* You have set up a credential, if necessary. 
+For more information, refer to the link:https://docs.ansible.com/automation-controller/latest/html/userguide/credentials.html[Credentials]
+section.
+* You have pushed a decision environment image to an image repository or you chose to use the provided image de-supported provided at link:http://registry.redhat.io/[registry.redhat.io].
```

Comment:
```suggestion
* You have pushed a decision environment image to an image repository or you chose to use the provided image de-supported provided at link:http://registry.redhat.io/[registry.redhat.io].
```
Is de-supported the image name?  I'd suggest enclosing it in backticks if it is.